gatsby-config.js

```diff
@@ -1,4 +1,20 @@
 const path = require(`path`)
+let targetJournal = 31;
+
+const findSearchData = (node, entry) => {
+
+  var data = ""
```

Comment:
`let` and `const` should always be used over `var` in modern javascript which enable identifying variables that should be const and provides better scoping.